unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentView.cs

```diff
@@ -170,13 +170,20 @@ public class ExploreV2MenuComponentView : BaseComponentView, IExploreV2MenuCompo
 
     internal RectTransform profileCardRectTranform;
     internal RealmSelectorComponentView realmSelectorModal;
+    internal HUDCanvasCameraModeController hudCanvasCameraModeController;
 
     public override void Awake()
     {
         base.Awake();
 
         profileCardRectTranform = profileCard.GetComponent<RectTransform>();
         realmSelectorModal = ConfigureRealmSelectorModal();
+        hudCanvasCameraModeController = new HUDCanvasCameraModeController(GetComponent<Canvas>(), DataStore.i.camera.hudsCamera);
```

Comment:
You still need to subscribe to the `DataStore.i.camera.hudsCamera.OnChange` that might not be ready when you are constructing the View. That's the reason you need a reactive controller and instead of polluting the `BaseComponentView` with all the subscription to the event and such I find more clean to have it in a new controller and just use composition.